### PR TITLE
FDN-2397:upgrade scala v2.13.13 and sbt version upgrade & Scoverage

### PR DESCRIPTION
**Pull request by abhidwivedi2701**
In this PR we are upgrading scala version 2.13.13 and sbt as well as scoverage upgrade. For this we have to add type annotation in mentioned files:

 api/app/controllers/Addresses.scala
 api/app/controllers/CountryDefaults.scala
 api/app/controllers/Healthchecks.scala
 api/app/controllers/Timezones.scala
 api/test/utils/CollectionsSpec.scala
 api/test/utils/DigitalElementSpec.scala

### CODE REVIEW

**Comment by flow-tech** [comment]
helm diff detected no changes for location

File: api/test/utils/DigitalElementSpec.scala [hunk 1]
```diff
@@ -19,14 +19,14 @@ class DigitalElementSpec extends AnyWordSpec with Matchers {
...
-      .mkString(fieldDelimiter.toString)
-      .getBytes(),
-  )
+  ): DigitalElementIndexRecord = {
+    val bytes: Array[Byte] =
+      ArrayBuffer[Serializable](rangeStart, rangeEnd, country, region, city, latitude, longitude, postalCode)
+        .map(_.toString)
+        .mkString(fieldDelimiter.toString)
+        .getBytes()
+    DigitalElementIndexRecord(rangeStart, rangeEnd, fieldDelimiter, bytes)
```
[the remark this answers is not in the document]
Comment (gheine) [reply api/test/utils/DigitalElementSpec.scala:28]: let's revisit this together...

**Review by gheine — CHANGES_REQUESTED** [review]
see above

**Comment by flow-tech** [comment]
helm diff detected no changes for location

**Comment by flow-tech** [comment]
Click here for code coverage report: https://jenkins.flo.pub/job/flowcommerce/job/location/job/PR-528/2/scoverage-report/

**Pull request merged**